packages/test/local-server-tests/src/test/documentDirty.spec.ts

```diff
@@ -248,6 +248,9 @@ describe("Document Dirty", () => {
 
                 checkDirtyState("after value set", true, 0);
 
+                // Manually flush the ops so that they are sent as a batch.
+                (dataObject.context.containerRuntime as IContainerRuntime).flush();
```

Comment:
Wondering why this is needed?  Would be good to add an explaination in the comment.